docs/architecture/adr-002-ibc-relayer.md

```diff
@@ -4,19 +4,27 @@
 * {date}: {changelog}
 
 ## Definitions
+These definitions are specific for this document and they may not be consistent with the IBC Specification.
+
+IBC datagram - the transaction payload sent by the relayer over the physical network that includes client, connection, channel and IBC packet data. 
```

Comment:
A small clarification would make this definition amazing, namely: Where does the relayer send an IBC datagram to? i.e., what is the physical network endpoint delivering and handling an IBC datagrams from a relayer? Is this is a full node, for instance in Tendermint accepted through the `broadcast_tx_commit` API?